### ISSUE TITLE
torch.linalg.lstsq doc arguments error

### ISSUE THREAD

**Issue by cheyennee** [pytorch/pytorch#99316]
### 📚 The doc issue

According to [doc](https://pytorch.org/docs/stable/generated/torch.linalg.lstsq.html), the arguments of torch.linalg.lstsq are `A`、`B`、`rcond`、`driver`. But when using these arguments, below code will throw exception: `ERROR:linalg_lstsq() missing 2 required positional argument: "input", "b"`. Actually the arguments should be `input`、`b`、`rcond`、`driver`.
```
import torch
arg_1 = torch.rand([5, 3], dtype=torch.float32)
arg_2 = torch.rand([5, 3], dtype=torch.float32)
results = torch.linalg.lstsq(A=arg_1,B=arg_2,)
print(results)
```

**Comment by cheyennee** [comment]
Yeah. Otherwise, users who encounter this problem can only read the document and compare with error message, which is not user-fridendly.

**Comment by lezcano** [comment]
The issue here is that changing these APIs is often BC-breaking. I am not sure if changing the name of positional args in ATen is BC breaking though. Perhaps @ezyang knows?

**Comment by ezyang** [comment]
Changing the arg name in schema is bc breaking, but obviously we can fix the error messages pretty easily
